_dashboards/workspace/workspace-acl.md

```diff
@@ -151,3 +151,47 @@ To remove several collaborators simultaneously, follow these steps:
 2. Select the **Delete x collaborators** button.
 3. Review the confirmation modal that appears.
 4. Select **Confirm** to remove all selected collaborators from the table.
+
+## Configuring workspace privacy
+
+When permission control is enabled, workspace administrators can set who has access to a workspace, with the following three access levels:
+
+* **Private to collaborators (Default):** Gives only workspace collaborators can access the workspace. 
+* **Anyone can view:** Grants **Read only** permissions to all users in the workspace, allowing them to view workspace assets. 
+* **Anyone can edit:** Grants **Read and write** permissions to all users, allowing view, create, and update workspace assets.
+
+Collaborators are granted higher permissions when their individual access level differs from that set in the workplace settings. For example, if workspace privacy is set to “Anyone can edit”, any collaborator with read-only access level will also be able to edit the workspace assets.
+
+You can set up workspace privacy on the **Create Workspace** page as a **Dashboard admin**. You can also modify it on the **Collaborators** and **Workspace details** pages, as a **Workspace admin** or **Dashboard admin**.
+
```

Comment:
Confirm that the title of the "Create Workspace" page uses title case.

Reply:
Its the other way. I adjusted it.